cmake/codecheck/rules/deprecated_c_header.py

```diff
@@ -36,11 +39,13 @@
 allowed = [
     '#include <cassert>',
     '#include <ccomplex>',
+    '#include <cctype>',
     '#include <ctype>',
     '#include <cerrno>',
     '#include <cfenv>',
     '#include <cfloat>',
     '#include <cinttypes>',
+    '#include <cstdio>',
```

Comment:
`<cstdio>` is already on the list (line 59).

Reply:
Spam